webserver/predictor.py

```diff
@@ -14,6 +17,40 @@ def from_utc(utc_time: str) -> datetime.datetime:
     ).astimezone(timezone("Europe/Berlin")).replace(tzinfo=None)
 
 
+def train_model(self, train_x, train_y, **model_parameters):
```

Comment:
@McToel ist des ok, dass ich hier auch noch des train reingepackt habe?
Des passt mMn eingentlich ziehmlich gut da rein, so hat man viele Webseiten ML Sachen in einem File/Classe